scraper/smashgg.py

```diff
@@ -134,22 +136,46 @@ def get_smashgg_matches(self):
                 if loser_id is None:
                     continue
 
-                round_name = match.get("fullRoundText", None)
-                round_num = match.get("round", None)
-                best_of = match.get("bestOf", None)
+                # CHECK FOR A BYE OR A DQ
+                if loser_id is None:
+                    continue
+                else:
+                    entrant1_score = match['entrant1Score']
```

Comment:
maybe use match.get('entrant1Score', None) here or wrap this in a try/except block; otherwise the scraper will crash if the match object returned by the api doesn't have the 'entrant1Score' key for some reason